### PR TITLE
Dlg convert columns

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers This is ready for review.

Changes made:
- Added new radio button "Default" for if the user does not want to override on "Factor Options"
- Tab order
- Designer Layout
- Code layout

### CODE REVIEW

**Comment by stevekogo** [comment]
@africanmathsinitiative/developers Someone should review this. These are minor changes.

**Comment by AlexSananka** [comment]
@stevekogo I am testing this

File: instat/dlgConvertColumns.vb [hunk 1]
```diff
@@ -110,10 +108,8 @@ Public Class dlgConvertColumns
         ucrBase.clsRsyntax.SetBaseRFunction(clsDefaultFunction.Clone())
```
Comment (AlexSananka) [inline instat/dlgConvertColumns.vb]: Why do we need.clone here?

Comment (lilyclements) [inline instat/dlgConvertColumns.vb]: Good point, I'll sort that out. Thanks!

**Comment by lilyclements** [comment]
@AlexSananka I have made the necessary changes, this is ready for a re-review

**Comment by AlexSananka** [comment]
@dannyparsons This is ready for you to review

**Pull request merged**